likelihood/testrun.py

```diff
@@ -29,3 +33,66 @@
 pf = {k: bf[k] for k in p0.keys()}
 print("Final params: ")
 print(pf)
+
+#======================DETERMINE ERRORS ON PARAMETERS========================
+
+# remove cobaya_out directory (just for now!) to make running code easier
+os.system('rm -r cobaya_out')  
+
+h = 0.005  # stepsize
+
+def fstep(params,signs):  # Determine likelihood at new steps 
```

Comment:
If params can only have a pair of values, I'd call just call it param1, and param2. Especially without docstring, one would naively think it can take any number of parameters. 
Perhaps better to create a small Fisher object that would just initialize with everything it needs internall.